CS410-Assignment2/src/sensorAdapter/SensorAdapter.java

```diff
@@ -0,0 +1,13 @@
+package sensorAdapter;
+
+import java.awt.Color;	//	for color associated with status
+
+public interface SensorAdapter {
+	
+	public double getSensorValue();	//	Value of Sensor
+	public String getSensorStatus();	//	Status of Sensor
+	public String getSensorName();	//	Name of Sensor
+	public Color getBarColor();	//	Color of bar based on status
+	public int getBarProgress();	//	Size of bar
+
+}

```

Comment:
This seems perfect to me, although commenting "size of bar" would be somewhat confusing for someone who didn't know it was going to be used with a JProgressBar (but since these are adapted for an awt GUI, it probably is fine)

Reply:
I probably should extend the comment to include the range from 0 to 100 that it uses.